examples/aes-ccm/src/ccm_mode.rs

```diff
@@ -0,0 +1,197 @@
+use std::convert::TryInto;
```

Comment:
This import isn't legal in hacspec.

Reply:
This is an issue I was also facing while implementing `aes-ccm`. I had number variables of different types and wasn't able to do operations on two different types without `try_into`. I've used it 19 times in `ccm_mode.rs` and I have no idea how to fix it. I'll need some help with this.

Reply:
Ok, I don't think you'll need it. When we have all the other issues in here resolved this will not be necessary anymore.